internal/crd-watch/crd_watch_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package crd_watch_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestCrdWatch(t *testing.T) {
```

Comment:
Thanks for noticing!
Can you move this function to `crd-watch_test.go`, because there is only one test file in the folder?

Reply:
/close

ACK will do, I'll likely fix this within https://github.com/kubevirt/ssp-operator/pull/604 now, was just worried at the time that there might be other examples in the codebase but I think we are okay. Closing this out.